src/code42cli/cmds/legal_hold.py

```diff
@@ -51,7 +69,10 @@ def legal_hold(state):
 
 
 @legal_hold.command()
-@matter_id_option
+@matter_id_option(
+    True,
+    "Identification number of the legal hold matter the custodian will be added to.",
```

Comment:
@annie-payseur When you have chance, could you help review these option help texts? I will tag you in spots.